talk/concurrency/condition.tex

```diff
@@ -2,8 +2,9 @@
 
 \begin{frame}[fragile]
   \frametitlecpp[11]{Condition variables}
-  \begin{block}{Communicating thread dependencies}
+  \begin{block}{Communicating between threads}
     \begin{itemize}
+      \item Imagine the case where threads are waiting for work from another thread
```

Comment:
```suggestion
      \item case where threads are waiting for (an)other thread(s)
```
That way it should fit on the line and look better

Reply:
I took it into account, but changed it a bit more. 🙂